sys-process/cpu-x/cpu-x-3.2.4.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils xdg
+
+MY_PN='CPU-X'
+DESCRIPTION='A Free software that gathers information on CPU, motherboard and more'
+HOMEPAGE='https://x0rg.github.io/CPU-X'
+SRC_URI="https://github.com/X0rg/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE='GPL-3+'
+SLOT='0'
+KEYWORDS='-* ~amd64'
+IUSE='
+	+bandwidth check-update +dmidecode force-libstatgrab +gtk ignore-staticlibs
```

Comment:
I think `ignore-staticlibs` should just be called `static-libs` like it is in the current Gentoo tree. Just tune the logic so if it's disabled, it disables building static libraries here as well.

Reply:
`ignore-staticlibs` "builds **portable** binary without static libraries".